.gitmodules

```diff
@@ -1,7 +1,7 @@
 [submodule "go-ethereum"]
 	path = go-ethereum
 	url = https://github.com/EspressoSystems/geth-arbitrum.git
-	branch = integration
+	branch = jh/verifier-address
```

Comment:
can we push this to a celestia-integration branch?

Reply:
We are using the same `go-ethereum`. So it seems unnecessary to create another branch in go-ethereum